lib/galaxy/datatypes/converters/interval_to_bgzip_converter.xml

```diff
@@ -29,27 +38,49 @@
     <tests>
         <test>
             <param name="input1" ftype="bed" value="droPer1.bed"/>
-            <output name="output1" ftype="bgzip" value="droPer1.bgzip"/>
+            <output name="output1" ftype="bgzip" value="droPer1.bgzip" decompress="true">
+                <expand macro="assert_size" value="131"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n -k3,3n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="encodepeak" value="encode.broad.peak"/>
             <output name="output1" ftype="bgzip">
-                <assert_contents>
-                    <has_size value="299" delta="10"/>
-                </assert_contents>
+                <expand macro="assert_size" value="299"/>
             </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n -k3,3n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="gff" value="gff_filter_by_feature_count_out2.gff"/>
-            <output name="output1" ftype="bgzip" value="bgzip_filter_by_feature_count_out2.bgzip"/>
+            <output name="output1" ftype="bgzip" value="bgzip_filter_by_feature_count_out2.bgzip" decompress="true">
+                <expand macro="assert_size" value="638"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k4,4n"/>
+                <has_text text="-k5,5" negate="true"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="interval" value="2.interval"/>
-            <output name="output1" ftype="bgzip" value="2.bgzip"/>
+            <output name="output1" ftype="bgzip" value="2.bgzip" decompress="true">
+                <expand macro="assert_size" value="208"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n -k3,3n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="vcf" value="vcf_to_maf_in.vcf"/>
-            <output name="output1" ftype="bgzip" value="bgzip_to_maf_in.bgzip"/>
+            <output name="output1" ftype="bgzip" value="bgzip_to_maf_in.bgzip" decompress="true">
+                <expand macro="assert_size" value="707"/>
+            </output>
+            <assert_command>
+                <has_text text="-k1,1 -k2,2n"/>
+            </assert_command>
         </test>
         <test>
             <param name="input1" ftype="gtf" value="cufflinks_out1.gtf"/>
```

Comment:
I think it would be good to have the output for this one also (see https://github.com/galaxyproject/galaxy-test-data/pull/24).

Reply:
Do you thing that the zip output is needed here? How do you want to compare? Using `decompress="true"`, or just `sim_size`?

Reply:
I think sim_size is ok.

Reply:
Which is equivalent to the `<has_size value="270" delta="10" />` assertion - which we already have :) 

So I guess this PR is finished, or? I could not spot anything missing.

Reply:
Yes I think it is ok. I will cancel my PR on galaxy-test-data as we don't really need it.